cvat-ui/src/components/project-page/project-page.tsx

```diff
@@ -28,18 +28,16 @@ export default function ProjectPageComponent(): JSX.Element {
     const id = +useParams<ParamType>().id;
     const dispatch = useDispatch();
     const history = useHistory();
-    const projects = useSelector((state: CombinedState) => state.projects.current);
+    const projects = useSelector((state: CombinedState) => state.projects.current.map((project) => project.instance));
```

Comment:
We should not use .map in useSelector. It creates unnecessary rerenders because useSelector always return new object.